validation/genesis/validation-inputs/34443/meta.toml

```diff
@@ -0,0 +1,14 @@
+genesis_creation_commit = "3eda4cd594ba6409584eb6ef95c341d98419e392"
+monorepo_build_command = "yarn install --no-frozen-lockfile"
+genesis_creation_command = """\
```

Comment:
Perhaps this could be changed to `contracts_file_type` and the valid values are just `["single", "multiple"]` (indicating single .deploy file or one .json file for every contract). I think the other variables involved in this command like the `chainID` could be derived by the go code and passed into this shell script